docs/business.yaml

```diff
@@ -718,117 +662,178 @@ paths:
-                voluntary-dissolution-failed-missing-filing-name-response:
-                  summary: Voluntary Dissolution - Missing Filing Name Response
+                    rootCause: errors:[error:Share class Common Shares must provide value for maximum number of shares,path:/filing/alteration/shareClasses/0/maxNumberOfShares/]
+                alteration-failed-name-request-does-not-have-the-same-legal-name-response:
+                  summary: Alteration - Name Request Does Not Have the Same Legal Name Response
                   value:
                     errorMessage: API backend third party service error.
-                    rootCause: errors:[message:filing/header/name is a required property]
-                agm-extension-failed-invalid-total-approved-extension-response:
-                  summary: AGM Extension - Invalid Total Approved Extension Response
+                    rootCause: errors:[error:Alteration of Name Request has a different legal name.,path:/filing/alteration/nameRequest/legalName]
+                alteration-failed-name-request-does-not-have-the-same-legal-type-response:
+                  summary: Alteration - Name Request Does Not Have the Same Legal Type Response
                   value:
                     errorMessage: API backend third party service error.
-                    rootCause: errors:[error:Fail to grant extension.],filing:agmExtension:agmDueDate:2026-06-20,agmYear:2024,alreadyExtended:false,currentDate:2024-07-10,expireDateApprovedExt:2026-06-20,extReqForAgmYear:false,extensionDuration:6,incorporationDate:2024-06-20T17:31:58.000Z,isEligible:true,isFirstAgm:true,isGoodStanding:true,isPrevExtension:false,prevAgmDate:null,prevExpiryDate:null,requestExpired:false,totalApprovedExt:7,year:2024,business:foundingDate:2024-06-20T17:31:58.000+00:00,identifier:BC0882365,legalName:0882365 B.C. LTD.,legalType:BC,header:certifiedBy:APISpecs,date:2024-07-10,name:agmExtension
-                change-of-directors-failed-missing-business-response:
-                  summary: Change Of Directors - Missing Business Response
+                    rootCause: errors:[error:Name Request legal type is not same as the business legal type.,path:/filing/alteration/nameRequest/legalType]
+                alteration-failed-name-request-is-not-approved-response:
+                  summary: Alteration - Name Request Is Not Approved Response
                   value:
                     errorMessage: API backend third party service error.
-                    rootCause: errors:[message:A valid business is required.]
+                    rootCause: errors:[error:Alteration of Name Request is not approved.,path:/filing/alteration/nameRequest/nrNumber,error:Alteration of Name Request has a different legal name.,path:/filing/alteration/nameRequest/legalName]
+                change-of-directors-invalid-address-country:
```

Comment:
consider changing it to `change-of-directors-failed-invalid-address-country`

Reply:
Thx and updated.